desktop/main.dev.ts

```diff
@@ -132,7 +132,11 @@ const createBrowserView = () => {
 };
 
 const addIpcEventListeners = () => {
-  ipcMain.handle(ipcConsts.GET_OS_THEME_COLOR, () => nativeTheme.shouldUseDarkColors);
+  ipcMain.handle(ipcConsts.GET_OS_THEME_COLOR, () => isDarkMode);
```

Comment:
Possible bug here:
1. `isDarkMode` variable was set on startup
2. Then someone toggled this flag (via `SET_THEME_COLOR`)
3. Then someone calls `GET_OS_THEME_COLOR` — it will return an outdated (falsy) value

So
```suggestion
  ipcMain.handle(ipcConsts.GET_OS_THEME_COLOR, () => StoreService.get('userSettings.darkMode'));
```
And also you can get rid of `let isDarkMode: boolean = ...` since it is unused variable

Reply:
I think now you got the mistake. But you still left unused variable.